src/index.html

```diff
@@ -4,37 +4,50 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles/correct.css" />
     <link
       rel="stylesheet"
-      href="https://unicons.iconscout.com/release/v4.0.0/css/line.css"
+      href="https://unicons.iconscout.com/release/v3.0.1/css/line.css"
     />
-    <link rel="stylesheet" href="./style.css" />
     <title>Document</title>
   </head>
   <body>
-    <h1>My Todo List</h1>
     <div id="todo-list">
-      <form action="form">
-        <div class="form-div">
-          <input type="text" class="todo-input" value="Demo" />
-          <button class="todo-button" type="submit">
-            <i class="uil uil-rotate-360"></i>
-          </button>
+      <form action="">
+        <div class="choose">
+          <select name="todos" class="filter-tasks">
+            <option value="all" class="active">All</option>
+            <option value="completed">Completed</option>
+          </select>
         </div>
-        <div class="form-div">
-          <input
-            type="text"
-            class="add-todo"
-            placeholder="Add to your list..."
-          />
+        <div class="input-div">
+          <div class="input-div-one">
+            <div>
+              <input type="text" class="demo" value="Demo" />
+              <button class="todo-button" type="submit">
+                <i class="uil uil-refresh"></i>
+              </button>
+            </div>
+          </div>
+          <div class="input-div-two">
+            <input type="text" class="todo-input" value="" />
```

Comment:
- [ ] I can see that you have two input fields in your app whose functionality is not defined because I could only add just one item while using the specified text fields. Kindly check it out 👍 

![image](https://user-images.githubusercontent.com/63473584/179373075-3a15cf2b-9aa6-48ec-b5e8-775ee4037f34.png)